client/src/main/java/org/glassfish/tyrus/client/SslContextConfigurator.java

```diff
@@ -578,6 +578,17 @@ public SSLContext createSSLContext() {
             TrustManagerFactory trustManagerFactory = null;
             KeyManagerFactory keyManagerFactory = null;
 
+            if ("SunMSCAPI".equals(System.getProperty(TRUST_STORE_PROVIDER))) {
```

Comment:
@lwahonen Can instead of this code block be just updated the row below:

`if (trustStoreBytes != null || trustStoreFile != null) {`

with `if (trustStoreBytes != null || trustStoreFile != null || "SunMSCAPI".equals(trustStoreType)) {` ?